content/posts/mongo_db_databases.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Introducing MongoDB databases"
+date: 2024-10-14
+tags:
+    - News
+author: Costin Sin
+linkedIn: https://www.linkedin.com/in/costin-sin/
+thumbnail: /images/ferretdb-genezio.webp
+preview: Genezio users can now set up MongoDB databases! You can now easily manage your MongoDB databases alongside your serverless functions. Learn more about this new feature.
+# meta data start
+description: Learn about the new MongoDB database feature in Genezio. Easily set up and manage MongoDB databases alongside your serverless functions.
+meta_og_url: "https://genezio.com/blog/mongo_db_databases/"
+meta_og_image: "https://genezio.com/images/ferretdb-genezio.webp"
+# meta data end
+customHeader: "White header"
+customFooter: "White footer"
+readTime: 10
+url: /blog/mongo_db_databases/
+---
+
+Hello everyone! We’re excited to share some big news: Genezio users can now set up MongoDB databases! Thanks to our new partnership with MongoDB Atlas, you can easily manage your MongoDB databases alongside your serverless functions. This means you get a powerful, fully managed database service that’s easy to use, scalable, and secure.
```

Comment:
Hey developers! We’ve got some exciting news—Genezio now supports MongoDB Atlas! As a developer, I know how important it is to have a reliable, scalable database, and now you can easily set up and manage NoSQL databases right from the Genezio platform.